.vscode/settings.json

```diff
@@ -1,19 +0,0 @@
-{
-  "[javascript]": {
-    "editor.insertSpaces": true,
-    "editor.tabSize": 2
-  },
-  "[typescript]": {
-    "editor.insertSpaces": true,
-    "editor.tabSize": 2
-  },
-  "[json]": {
-    "editor.insertSpaces": true,
-    "editor.tabSize": 2
-  },
-  "javascript.format.insertSpaceAfterFunctionKeywordForAnonymousFunctions": false,
-  "typescript.format.insertSpaceAfterFunctionKeywordForAnonymousFunctions": false,
-  "files.trimTrailingWhitespace": true,
-  "files.insertFinalNewline": true,
-  "files.trimFinalNewlines": true
-}
```

Comment:
I saw these configuration are moved into Tedious.code-workspace.  Is settings.json becoming a outdated way to handle this or it is just simpler to do this within the Tedious.code-workspace?

Reply:
settings.json is fully supported and not going away to the best of my knowledge. 

I believe it is simpler as you can centralize more configurations in the workspace config. It technically gives you more capabilities as settings.json relates only to a single set of configurations for the folder, the configuration of the workspace allows for more options and integrations and I believe it also has codespace support.
Some examples are folder exclusions on the workspace to hide stuff that is not necessary, this is not supported on settings.json

Reply:
You can also include extension recommendations in the same file. 
It is also better for monolithic projects.